tests/performance/pom.xml

```diff
@@ -155,6 +155,7 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
+                            <argLine>-Djmh.ignoreLock=true</argLine>
```

Comment:
We want this config on this profile. This shoul only be set for the default profile.